extensions/common/iam/oauth2/oauth2-daps/src/test/java/org/eclipse/edc/iam/oauth2/daps/Rfc8707IntegrationTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Amadeus - initial API and implementation
+ *       Microsoft Corporation - Use IDS Webhook address for JWT audience claim
+ *
+ */
+
+package org.eclipse.edc.iam.oauth2.daps;
+
+import org.eclipse.edc.junit.annotations.ComponentTest;
+import org.eclipse.edc.junit.extensions.EdcExtension;
+import org.eclipse.edc.jwt.spi.JwtRegisteredClaimNames;
+import org.eclipse.edc.policy.model.Policy;
+import org.eclipse.edc.spi.iam.IdentityService;
+import org.eclipse.edc.spi.iam.TokenParameters;
+import org.eclipse.edc.spi.iam.VerificationContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+
+import java.nio.file.Path;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.edc.junit.testfixtures.TestUtils.findBuildRoot;
+
+@ExtendWith(EdcExtension.class)
+@ComponentTest
+public class Rfc8707IntegrationTest {
```

Comment:
Please don't use an integration test for this as they slow down the build. Unit tests should cover this.

Reply:
I have made this test because I found that the `DapsIntegrationTest` test fail if in used with the feature flag of this PR set to true. This test is to demonstrate the concomitance used of the Daps extension with the new feature but with a correct configuration. 
I don't know why someone would use both at the same time so if you think, it is not needed, I can remove it. The new feature is already covert by a updated unit test included in this PR.

Reply:
Yes, please delete it. DAPs will be removed at some point as well as it is not a production-grade identity provider.